src/library/scala/StringContext.scala

```diff
@@ -56,11 +56,10 @@ import scala.annotation.tailrec
  */
 case class StringContext(parts: String*) {
 
-  import StringContext._
+  import StringContext.{checkLengths => scCheckLengths, glob, standardInterpolator => scStandardInterpolator}
 
   @deprecated("use same-named method on StringContext companion object", "2.13.0")
-  def checkLengths(args: scala.collection.Seq[Any]): Unit =
-    StringContext.checkLengths(args, parts)
+  def checkLengths(args: scala.collection.Seq[Any]): Unit = scCheckLengths(args, parts)
```

Comment:
What's the warning here? Scalac gets more opinionated every release, although I really should know these things.

Reply:
"Could be a one-liner" under `-Xlint:oneliners`. I wanted to simplify the import that was formerly unused.